FirebaseMessaging.podspec

```diff
@@ -36,10 +36,7 @@ device, and it is completely free.
     'Firebase/InstanceID/Public/*.h',
     'FirebaseInstallations/Source/Library/Private/*.h',
   ]
-  s.requires_arc = [
```

Comment:
curious: this is not needed?

Reply:
If the [reason mentioned by Paul](https://github.com/firebase/firebase-ios-sdk/pull/7617#discussion_r583771492) is the only one, than I think we don't need it any more.